tests/sentry/web/frontend/test_react_page.py

```diff
@@ -155,6 +155,40 @@ def test_redirect_to_customer_domain(self):
             assert response.status_code == 200
             assert response.redirect_chain == [(f"http://{org.slug}.testserver/issues/", 302)]
 
+    def test_doest_notredirect_to_customer_domain_for_unsupported_paths(self):
+        user = self.create_user("bar@example.com")
+        org = self.create_organization(owner=user)
+        self.login_as(user)
+
+        with self.feature({"organizations:customer-domains": True}):
+
+            url_name = "sentry-organization-create"
+            url_name_is_non_customer_domain = any(
+                fnmatch(url_name, p) for p in NON_CUSTOMER_DOMAIN_URL_NAMES
+            )
+            assert url_name_is_non_customer_domain
```

Comment:
```suggestion
            assert (
                url_name_is_non_customer_domain, 
                "precondition missing. org-create should be non-customer-domain"
            )
```

Might help future us 🤷